includes/discovery/toner.inc.php

```diff
@@ -31,7 +31,7 @@
                     $capacity_oid = ".1.3.6.1.2.1.43.11.1.1.8.1.$index";
                 }
 
-                $descr = trim(str_replace("\n", '', str_replace('"', '', snmp_get($device, $descr_oid, '-Oqv'))));
+                $descr = trim(str_replace("\n", '', preg_replace('/[^ \w]+/', '', snmp_get($device, $descr_oid, '-Oqv'))));
```

Comment:
it allow the spaces and words, removing any special chars because : / are making rrdtool throw an error, like in #4464